city_scrapers/spiders/det_regional_convention.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime, time
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class DetRegionalConventionSpider(CityScrapersSpider):
+    name = "det_regional_convention"
+    agency = "Detroit Regional Convention Facility Authority"
+    timezone = "America/Detroit"
+    start_urls = ["http://www.drcfa.org/upcoming-meetings"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+        for item in response.xpath("//div[@class='link']/ul/li/a"):
+            self._ignore = False
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            if self._ignore is False:
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Boards of directors of Detroit Regional \
```

Comment:
"Board of Directors" would be fine, but we'll likely need to check this based on the archived page